ethers-ext/example/transactions/Basic_20_TxTypeAccountUpdate.js

```diff
@@ -1,5 +1,4 @@
 const ethers = require("ethers");
-import { crypto } from "@ethersproject/crypto";
 const { Wallet } = require("../../dist/src/ethers"); // require("@klaytn/sdk-ethers");
```

Comment:
I mean it needs to be replaced to `require("@klaytn/ethers-ext")`

Reply:
Thanks.